pylsl/pylsl.py

```diff
@@ -1179,24 +1179,40 @@ def resolve_stream(*args):
 # === Module Initialization Code ===
 # ==================================
 
-# find and load library
-os_name = platform.system()
-bitness = 8 * struct.calcsize("P")
-if os_name in ['Windows', 'Microsoft']:
-    libname = 'liblsl32.dll' if bitness == 32 else 'liblsl64.dll'
-elif os_name == 'Darwin':
-    libname = 'liblsl32.dylib' if bitness == 32 else 'liblsl64.dylib'
-elif os_name == 'Linux':
-    libname = 'liblsl32.so' if bitness == 32 else 'liblsl64.so'
-else:
-    raise RuntimeError("unrecognized operating system:", os_name)
-libpath = os.path.join(os.path.dirname(__file__), 'lib', libname)
-if not os.path.isfile(libpath):
-    libpath = util.find_library('lsl' + str(bitness))
-if not libpath:
-    raise RuntimeError("library " + libname + " was not found - make sure "
+def find_liblsl_libraries():
+    # find and load library
+    os_name = platform.system()
+    if os_name in ['Windows', 'Microsoft']:
+        libsuffix = '.dll'
+    elif os_name == 'Darwin':
+        libsuffix = '.dylib'
+    elif os_name == 'Linux':
+        libsuffix = '.so'
+    else:
+        raise RuntimeError("unrecognized operating system:", os_name)
+
+    if 'PYLSL_LIB' in os.environ:
+        yield os.environ['PYLSL_LIB']
+
+    libbasepath = os.path.join(os.path.dirname(__file__), 'liblsl')
```

Comment:
It's only recently that the wheels started putting the dll's in the liblsl subfolder. Before that they were next to the pylsl.py. Is there a chance someone would use this new version of pylsl.py but still have the dll next to the file instead of in a subfolder?

Reply:
Only if they were manually copying their own binary into the `pylsl/lib` folder.

Reply:
On a second thought, `pylsl/lib` is the better place, so I fixed the search path and the error message